.github/workflows/base-windows.yml

```diff
@@ -0,0 +1,346 @@
+  build-quarkus:
+    name: Quarkus ${{ matrix.quarkus-version }} - OpenJDK11-${{ matrix.jdk }}
+    runs-on: ubuntu-latest
+    needs:
+      - get-quarkus-versions
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.get-quarkus-versions.outputs.build-matrix) }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: quarkusio/quarkus
+        fetch-depth: 1
+        ref: ${{ matrix.quarkus-version }}
+        path: quarkus
+    - uses: actions/cache@v2.1.5
+      with:
+        path: ~/.m2/repository
+        key: base-windows-${{ matrix.quarkus-version }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: base-windows-${{ matrix.quarkus-version }}-maven-
+    - name: Build quarkus
+      run: |
+        curl -L https://api.adoptopenjdk.net/v3/binary/latest/11/${{ matrix.jdk }}/linux/x64/jdk/hotspot/normal/openjdk -o jdk.tar.gz
+        export JAVA_HOME=$(pwd)/openjdk
+        echo ${JAVA_HOME}
+        mkdir -p ${JAVA_HOME}
+        tar xf jdk.tar.gz -C ${JAVA_HOME} --strip-components=1
+        ${JAVA_HOME}/bin/java -version
+        cd quarkus
+        ./mvnw ${COMMON_MAVEN_ARGS} -Dquickly
+    - name: Tar Maven Repo
+      run: tar -I 'pigz -9' -cf maven-repo-${{ matrix.quarkus-version }}-${{ matrix.jdk }}.tgz -C ~ .m2/repository
+    - name: Persist Maven Repo
+      uses: actions/upload-artifact@v2
+      with:
+        name: maven-repo-${{ matrix.quarkus-version }}-${{ matrix.jdk }}
+        path: maven-repo-${{ matrix.quarkus-version }}-${{ matrix.jdk }}.tgz
+    - name: Delete Local Artifacts From Cache
+      run: rm -r ~/.m2/repository/io/quarkus
+
+  native-tests:
+    name: Q ${{ matrix.quarkus-version }} - ${{ matrix.category }} - OpenJDK11-${{ matrix.jdk }}
+    needs:
+      - build-quarkus
+      - build-mandrel
+      - get-quarkus-versions
+    runs-on: windows-latest
+    # Ignore the following YAML Schema error
+    timeout-minutes: ${{matrix.timeout}}
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.get-quarkus-versions.outputs.tests-matrix) }}
+    steps:
+      - name: Download Maven Repo
+        if: startsWith(matrix.os-name, 'windows')
+        uses: actions/download-artifact@v1
+        with:
+          name: maven-repo-${{ matrix.quarkus-version }}-${{ matrix.jdk }}
+          path: .
+      - name: Extract Maven Repo
+        if: startsWith(matrix.os-name, 'windows')
+        shell: bash
+        run: tar -xzf maven-repo-${{ matrix.quarkus-version }}-${{ matrix.jdk }}.tgz -C ~
+      - name: Download Mandrel
+        if: startsWith(matrix.os-name, 'windows')
+        uses: actions/download-artifact@v1
+        with:
+          name: jdk-${{ matrix.jdk }}
+          path: .
+      - name: Extract Mandrel
+        if: startsWith(matrix.os-name, 'windows')
+        shell: bash
+        run: |
+          mkdir -p openjdk
+          tar -xzf jdk-${{ matrix.jdk }}.tgz -C openjdk --strip-components=1
+          openjdk/bin/native-image --version
```

Comment:
This seems to cause test failures. Shouldn't this take a mandrel build rather than a boot JDK?

Reply:
Hmm the issue seems related to trying to run a cmd file from a bash shell. I changes it to `java --version` like we do in https://github.com/graalvm/mandrel/blob/acedc5cca3bd740df3d4e3ef6df415ad9ef6fa47/.github/workflows/base-windows.yml#L354

Thanks for pointing this out!